### ISSUE TITLE
Sugar dep breaks 6to5!

### ISSUE THREAD

**Issue by mgcrea** [klei/ng-dependencies#4]
It breaks on the require of `gulp-angular-filesort`:

```
TypeError: Object 0 has no method 'call'
    at /Users/[...]/node_modules/gulp-angular-filesort/node_modules/ng-dependencies/node_modules/sugar/release/sugar-full.development.js:6126:12
```

Related issue on https://github.com/klei/gulp-angular-filesort/issues/20

It looks like sugar breaks 6to5! Overriding prototypes is never a good idea!

```
TypeError: Object 0 has no method 'call'
    at /Users/olivier/Projects/ng-tools/gulp-ng-channels/node_modules/gulp-angular-filesort/node_modules/ng-dependencies/node_modules/sugar/release/sugar-full.development.js:6126:12
    at Array.forEach (native)
    at extend.keys (/Users/olivier/Projects/ng-tools/gulp-ng-channels/node_modules/gulp-angular-filesort/node_modules/ng-dependencies/node_modules/sugar/release/sugar-full.development.js:6125:12)
    at /Users/olivier/Projects/ng-tools/gulp-ng-channels/node_modules/gulp-angular-filesort/node_modules/ng-dependencies/node_modules/sugar/release/sugar-full.development.js:179:25
    at Array.map (native)
    at Array.<anonymous> (/Users/olivier/Projects/ng-tools/gulp-ng-channels/node_modules/gulp-angular-filesort/node_modules/ng-dependencies/node_modules/sugar/release/sugar-full.development.js:180:23)
    at Object.<anonymous> (/Users/olivier/Projects/ng-tools/gulp-ng-channels/node_modules/gulp-6to5/node_modules/6to5-core/lib/6to5/traverse/scope.js:40:86)
    at Module._compile (module.js:456:26)
    at Object.Module._extensions..js (module.js:474:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:312:12)
    at Module.require (module.js:364:17)
    at require (module.js:380:17)
    at Object.<anonymous> (/Users/olivier/Projects/ng-tools/gulp-ng-channels/node_modules/gulp-6to5/node_modules/6to5-core/lib/6to5/traverse/index.js:7:16)
    at Module._compile (module.js:456:26)
    at Object.Module._extensions..js (module.js:474:10)
```

**Comment by mgcrea** [comment]
Would be fixed by https://github.com/klei/ng-dependencies/pull/5

**Comment by joakimbeng** [comment]
Fixed by #5
